rocAL/source/decoders/image/turbo_jpeg_decoder.cpp

```diff
@@ -28,7 +28,7 @@ THE SOFTWARE.
 
 TJDecoder::TJDecoder() {
     m_jpegDecompressor = tjInitDecompress();
-    if ((_scaling_factors = tj3GetScalingFactors(&_num_scaling_factors)) == NULL)
+    if ((_scaling_factors = tjGetScalingFactors(&_num_scaling_factors)) == NULL)
```

Comment:
Instead of hardcoding to a particular version, can we make detect the tjpeg version and make it work for both. Similarly what we do for FFMpeg

Reply:
tjGetScalingFactors is a common API present in all turbojpeg versions (from [v1.2.0](https://github.com/libjpeg-turbo/libjpeg-turbo/tree/1.2.0) to latest version). The tj3GetScalingFactors API is only introduced in the latest v3.x versions. tjGetScalingFactors function internally calls the tj3GetScalingFactors in the latest releases to get the scaling factors.